contributing/ci-release.txt

```diff
@@ -38,12 +38,24 @@ jobs should be listed under the :jenkinsview:`Release` view.
     -   * Generate the Bio-Formats Java downloads page
         * :jenkinsjob:`BIOFORMATS-5.0-release-downloads`
         * :jenkinsjob:`BIOFORMATS-5.1-release-downloads`
-    -   * Build the Bio-Formats C++ download artifacts
-        *
-        * :jenkinsjob:`BIOFORMATS-5.1-release-cpp`
     -   * Generate the Bio-Formats C++ downloads page
         *
-        * :jenkinsjob:`BIOFORMATS-5.1-release-cpp-downloads`
+        * :jenkinsjob:`BIOFORMATS-CPP-5.1-release-downloads`
+    -   * Tags the Bio-Formats SuperBuild source code repository
+        *
+        * :jenkinsjob:`BIOFORMATS-CPP-5.1-release-push-superbuild`
+    -   * Builds the release native C++ implementation for Bio-Formats
+        *
+        * :jenkinsjob:`BIOFORMATS-CPP-5.1-release`
+    -   * Builds the release native C++ implementation for Bio-Formats (Unix superbuild)
+        *
+        * :jenkinsjob:`BIOFORMATS-CPP-5.1-release-superbuild`
+    -   * Builds the release native C++ implementation for Bio-Formats ("Windows superbuild)
+        *
+        * :jenkinsjob:`BIOFORMATS-CPP-5.1-release-win-superbuild`
+    -   * Trigger the Bio-Formats C++ release jobs
```

Comment:
Minor comment, I would move the position of this line to the top of the Bio-Formats C++ rows in the jobs table to mirror what we currently have for OMERO and Bio-Formats.